packages/slice-machine/lib/models/common/widgets/Image/components.tsx

```diff
@@ -72,7 +72,11 @@ export const AddThumbnailButton: React.FC<{
           alignItems: "center",
         }}
       >
-        <BsFillPlusSquareFill size={34} color="#5263BA" />
+        <BsFillPlusSquareFill
+          size={34}
+          color="#5263BA"
+          aria-label="Add a thumbnail"
```

Comment:
aria-labels for selectors are aa good idea 👍

Reply:
Yep, double benefit of an accessibility check 😄